test/integration/setup.js

```diff
@@ -290,6 +292,7 @@ async function setup() {
     { address: ramm.address, code: 'RA' },
     { address: st.address, code: 'ST' },
     { address: coverProducts.address, code: 'CP' },
+    { address: coverOrder.address, code: 'LO' },
```

Comment:
This will only lead to confusion. Let's rename `CoverOrder` to `LimitOrder` (or even `LimitOrders` - plural)